avocado/plugins/runner_nrunner.py

```diff
@@ -67,6 +67,16 @@ def initialize(self):
                                  metavar="HOST:PORT",
                                  help_msg=help_msg)
 
+        help_msg = ('Buffer size that status server uses.  This should '
+                    'generally not be a concern to most most users, but '
+                    'it can be tunned in case a runner generates very large '
```

Comment:
Maybe replace this string with "...in case a test generates..."? (to be closer to users)

Reply:
How about:

"Buffer size that status server uses.  This should generally not be a
concern to most most users, but it can be tunned in case a runner
generates very large status messages, which is common if a test
generates a lot of output. Default is 33554432 (32MiB)"